admin/admin-menu-and-tabs.php

```diff
@@ -59,12 +59,11 @@ public function __construct() {
      * @since 0.1
      */
     public function register_menu() {
-        add_menu_page( __( 'Extensions (DT)', 'disciple_tools' ), __( 'Extensions (DT)', 'disciple_tools' ), 'manage_dt', 'dt_extensions', [ $this, 'extensions_menu' ], 'dashicons-admin-generic', 59 );
```

Comment:
Why was this removed?

Reply:
@prykon Because the theme already declares the menu. And since this version is slightly different than the theme version it results in a duplicate declaration.